src/components/State/Store.tsx

```diff
@@ -47,6 +47,15 @@ type Store = {
     showHiddenTags: boolean;
     setShowHiddenTags: (show: boolean) => void;
 
+    tags: { tagId: string, tagName: string, newValue: string }[];
```

Comment:
```suggestion
    tags: <new type name>;
```
make a type dec in a types file and import it